app/src/main/java/cm/aptoide/pt/app/view/AppViewPresenter.java

```diff
@@ -946,12 +940,14 @@ private void handleInstallButtonClick() {
   }
 
   private void showRecommendsDialog(boolean isLoggedIn, String packageName) {
-    if (isLoggedIn && appViewManager.shouldShowRecommendsPreviewDialog()) {
-      view.showRecommendsDialog();
-      appViewAnalytics.sendLoggedInRecommendAppDialogShowEvent(packageName);
-    } else if (!isLoggedIn && appViewManager.canShowNotLoggedInDialog()) {
-      appViewNavigator.navigateToNotLoggedInShareFragmentForResult(packageName);
-      appViewAnalytics.sendNotLoggedInRecommendAppDialogShowEvent(packageName);
+    if (appViewManager.shouldShowRecommendsDialogs()) {
```

Comment:
Why is this a separate method from the shouldShowPreview ? Personally i would add the same condition on the same method.